janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -4919,15 +4921,23 @@ private Vertex prepareDataForEdgePropertyFilterTest(){
 
     @Test
     public void simpleLogTest() throws InterruptedException {
-        simpleLogTest(false);
+        if (getStoreFeatures().isKeyOrdered()) {
+            simpleLogTest(false, false);
+        } else {
+            simpleLogTest(false, true);
+        }
     }
```

Comment:
Feels like we should always test for non-string id (for any storage backend) and test string ids only when `getStoreFeatures().isKeyOrdered()`. In such case, I would suggest to use `@ParameterizedTest` here and in `simpleLogTestWithFailure()` instead of skipping `long` vertex id test for any storage backend which supports String ids.
I.e. something like below:
```
@ParameterizedTest
@ValueSource(booleans = {true, false})
public void simpleLogTest(boolean useStringId) throws InterruptedException {
  if(useStringId && getStoreFeatures().isKeyOrdered()) return;
  simpleLogTest(false, useStringId);
}
```